config.yml

```diff
@@ -286,6 +286,9 @@ genesis:
         mint_allocation_supplier: 0.7
         mint_allocation_source_owner: 0.15
         mint_allocation_application: 0.0
+        # The dao reward address SHOULD match that of the "pnf" below (i.e. `make poktrolld_addr ACC_NAME=pnf`).
```

Comment:
Can you add a `TODO_MAINNET(@Olshansk): Consolidate the usage of DAO/PNF throughout the configs & codebase`